qiskit_experiments/curve_analysis/composite_curve_analysis.py

```diff
@@ -416,11 +421,11 @@ def _run_analysis(
                     analysis._create_curve_data(curve_data=formatted_subset, **metadata)
                 )
 
-            # Add extra column to identify the fit model
-            table["group"] = analysis.name
             curve_data_set.append(table)
 
-        combined_curve_data = pd.concat(curve_data_set)
+        combined_curve_data = ScatterTable.from_dataframe(
```

Comment:
You need to return this object and `fit_dataset` (or individual item in the dict) as ArtifactData.

Reply:
Thanks for catching this. Fixed in 2052a24.

Reply:
Oops, 2052a24 had an error but 120326f fixes it. `curve_data` contains the full multi-analysis `ScatterTable` while `fit_summary` is a dictionary of `CurveFitResult`s keyed by analysis, and loading works. Example `CrossResonanceHamiltonian` experiment: https://quantum.ibm.com/experiments/9f4b8f5f-dcad-40b2-88b3-351df2cdd379